docs/UserGuide.md

```diff
@@ -425,45 +416,36 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
 
 ### Approve leave record by index: `approve-leave`
 
-* **What It Does:**
-  * Approve a leave record by the given index
-  * **Format:**
-  * `approve-leave INDEX`
-* **Examples:**
+1. Get the [index](#glossary) of the leave under the [leave list](#glossary).
 
-| S/N | Command information                                                                                                                                                                     |
-|-----|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `approve-leave 1` <br><br> **Output:** Approve the leave record of index 1|
+> [!NOTE]
+> If the leave request is not found, consider using `find-all-leave` or any `find commands` to locate the leave request in the leave list.
 
-* **Acceptable Values:**
-  * The index should be a positive number starting from 1 and within the range of the leave list.
-* **Expected Output on Success:**
-  * GUI Changes: The approved leave will be marked as approved with green highlight.
-  * Message shown to user: "Approved Leave:” with the information of the approved leave
-* **Expected Output on Failure:**
-  * `Leave previously approved: `with the information of the previously approved leave - the leave given by the index was previously approved
-  * `The leave index provided is invalid:  - the index is out of bounds
+2. Type in the following [command](#glossary) in the [command box](#glossary) `approve-leave LEAVE_LIST_INDEX`
+3. Press "enter" on your keyboard and the specified leave request is approved.
+
+| Error message | Why it happens | Fix |
+|---------------|----------------|-----|
+| `Invalid command format!` | The command you input does not follow the specified format | Ensure the command you entered follows the following format: `approve-leave LEAVE_LIST_INDEX`.
+| `The leave index provided is invalid` | The index specified does not refer to any leave request | Double check if the index appears in the leave list. Alternatively, use [find-all-leave]() or any [find commands]() to locate the leave request in the leave list. Afterwards, use the correct leave index in the `approve-leave` command. |
+| `Leave previously approved: ` followed by the leave information | The provided leave request is already approved in HRMate | If this is the leave you would like to approve, you don't have to do anything.
```

Comment:
For follow up actions: would something like "Double check if the input index is correct as the leave specified is already approved" be better?